README.md

```diff
@@ -1,77 +1,58 @@
-# Crypto LD (Linked Data) _(crypto-ld)_
+# Cryptographic Key Pair Library for Linked Data _(crypto-ld)_
 
 [![Build Status](https://travis-ci.org/digitalbazaar/crypto-ld.png?branch=master)](https://travis-ci.org/digitalbazaar/crypto-ld)
 
-> A Javascript library for cryptographic operations using Linked Data
+> A Javascript library for generating Linked Data cryptographic key pairs
```

Comment:
Isn't it also about providing a common API for using those keys?

Reply:
Yeah. I'll edit to add that.

Reply:
Edited.